### ISSUE TITLE
Support parasitic metaprojects?

### ISSUE THREAD

**Issue by mlincett** [WIPACrepo/docker-icecube-icetray#11]
I am trying to build `realtime` starting from an `icetray-devel` image, however the `cmake` stage fails due to lack of `/usr/local/icetray/CMakeCache.txt`.

In a Slack thread, @blaufuss mentioned "the cvmfs installed builds are stripped of many of the needed items for a parasitic build".

Since I would like to have `realtime` containerised, I am not sure what is the best way forward.

Feel free to comment in the [realtime issue](https://github.com/icecube/realtime/issues/16) as well.

**Comment by mlincett** [comment]
Following-up a comment by @briedel on Slack: for the [Skymap Scanner Dockerfile](https://github.com/icecube/skymap_scanner/blob/main/Dockerfile) it seems we used to just copy over some python modules, rather than trying to compile `realtime`.

I have skimmed through the commit history and it seems that there was no coupling to realtime for the cloud version.

**Comment by blaufuss** [comment]
Correct:  only the GFU llh engine is a compiled part of the realtime project, everything else is scripts, etc

**Comment by mlincett** [comment]
> This could probably be adjusted so it does a normal build for the devel image

This sounds reasonable!
